app/controllers/miq_ae_class_controller.rb

```diff
@@ -1955,11 +1958,14 @@ def create_action_url(node)
 
   def get_rec_name(rec)
     column = rec.display_name.blank? ? :name : :display_name
-    if rec.kind_of?(MiqAeNamespace) && rec.domain? && (!rec.editable? || !rec.enabled)
-      add_read_only_suffix(rec, rec.send(column))
-    else
-      rec.send(column)
+    if rec.kind_of?(MiqAeNamespace) && rec.domain?
+      editable_domain = Tenant.current_tenant.editable_domains.include?(rec)
+      enabled_domain  = Tenant.current_tenant.enabled_domains.include?(rec)
```

Comment:
@mkanoor i used Tenant.current_tenant.enabled_domains method instead of record.enabled thinking that there was a use case behind adding that method. i don't see that method being used any where else, please let me know if i should remove usages of enabled_domains method and change it back to using record.enabled.